extensions/amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.out

```diff
@@ -0,0 +1,120 @@
+>>           ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:61:10 The tag 'div', a child tag of 'AMP-STORY-GRID-LAYER default', does not satisfy the reference point 'AMP-STORY-GRID-LAYER animate-in'. (see https://www.ampproject.org/docs/reference/components/amp-story) [AMP_TAG_PROBLEM]
+>>           ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:61:10 The attribute 'animate-in' may not appear in tag 'div'. [DISALLOWED_HTML]
+>>           ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:61:10 The attribute 'animate-in-after' may not appear in tag 'div'. [DISALLOWED_HTML]
+>>           ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:61:10 The attribute 'animate-in-duration' may not appear in tag 'div'. [DISALLOWED_HTML]
+>>           ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:61:10 The attribute 'scale-end' may not appear in tag 'div'. [DISALLOWED_HTML]
+>>           ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:61:10 The attribute 'scale-start' may not appear in tag 'div'. [DISALLOWED_HTML]
+|              animate-in="zoom-out"
+|              scale-start="2"
+|              scale-end="invalid"
+|              animate-in-duration="10s"
+|              class="img-container"
+|              animate-in-after="ken-burns-img1">
+|              <amp-img id="ken-burns-img2"
+>>             ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:68:12 The attribute 'animate-in-after' may not appear in tag 'amp-img'. (see https://www.ampproject.org/docs/reference/components/amp-img) [DISALLOWED_HTML]
+>>             ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:68:12 The attribute 'animate-in-duration' may not appear in tag 'amp-img'. (see https://www.ampproject.org/docs/reference/components/amp-img) [DISALLOWED_HTML]
+>>             ^~~~~~~~~
+amp-story/1.0/test/validator-amp-story-animations-visual-effects-parameters-error.html:68:12 The attribute 'translate-y' may not appear in tag 'amp-img'. (see https://www.ampproject.org/docs/reference/components/amp-img) [DISALLOWED_HTML]
+|                src="https://picsum.photos/1024/768?image=1026"
+|                layout="fixed"
+|                width="1024"
+|                height="768"
+|                translate-y="-200px"
+|                animate-in-duration="30s"
+|                animate-in-after="ken-burns-img1">
+|              </amp-img>
+|            </div>
+|          </div>
+|        </amp-story-grid-layer>
+|      </amp-story-page>
+|    </amp-story>
+|  </body>
+|  </html>
```

Comment:
Nit: new line (you can configure your software to add it automatically on save I think)

Reply:
Thanks. Yeah I have that enabled but I think whenever you pass the `--update_tests` in the validator it removes it.

/cc @Gregable as FYI